File: Jungwon/211216/숫자 게임.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

int solution(vector<int> A, vector<int> B) {
    int answer = 0, size = A.size();
    sort(A.begin(), A.end());
    sort(B.begin(), B.end());
    int i=0, j=0;
    while (i<size && j<size) {
        if (A[i] < B[j]) {
            i++;
            answer++;
        }
        j++;
    }
    return answer;
}

File: YuJin/211216/숫자게임.py
def solution(A, B):
    answer = 0 
    A.sort(reverse = True)
    B.sort(reverse = True)
    j = 0
    for element in B:
        for j in range(len(A)):
            if element > A[j]:
                answer += 1
                A.pop(j)
                break
    
    return answer
